Python_Engine/Compute/JupyterLab.cs

```diff
@@ -0,0 +1,47 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2019, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.IO;
+
+namespace BH.Engine.Python
+{
+    public static partial class Compute
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        public static void JupyterLab(int port = 7777, bool start = false)
+        {
+            string path = Path.Combine(Query.EmbeddedPythonHome(), "Scripts", "jupyter-lab.exe");
+            if (!File.Exists(path))
+                throw new FileNotFoundException("Cannot find jupyter lab", path);
+
+            path = Path.GetFullPath(path);
+            string configPath = Path.Combine(Query.EmbeddedPythonHome(), "Scripts", "jupyter_lab_config.py");
+            if (start)
+                RunCommandAsync($"cd ../../../../ && {path} --port {port} --config {configPath}", false);
```

Comment:
Is this relative path guaranteed to always work? Or is it better to let users specify a path with this as a default?

Reply:
The relative path will work on all Windows 10 machines for sure, but I like that! Just pushed!